README.md

```diff
@@ -403,46 +404,47 @@ in this case, just run `ssh-keygen -R "mail.example.org"` as recommended.
     chown echobot: -R /run/echobot
    ```
 
-4. Now, update DNS to the new IP addresses.
+4. Now, update DNS entries. 
 
-   If other servers try to deliver messages to your server they may fail,
-   but normally email servers will retry delivering messages
+   If other MTAs try to deliver messages to your chatmail domain they may fail intermittently,
+   as DNS catches up with the new site settings 
+   but normally will retry delivering messages
    for at least a week, so messages will not be lost.
 
-5. Finally, you can execute `cmdeploy run --ssh-host 13.12.23.42` to turn on chatmail on the new server.
-    Your users will be able to use the chatmail server as soon as the DNS changes have propagated.
-    Voilà!
+5. Finally, you can execute `cmdeploy run --ssh-host 13.12.23.42` to turn on chatmail on the new relay.
+   Your users will be able to use the chatmail site as soon as the DNS changes have propagated.
```

Comment:
```suggestion
   Your users will be able to use the chatmail relay as soon as the DNS changes have propagated.
```

Many users will not access the "site" at this point; mostly "site" is fine for the migration guide, as "site" also has this geographical connotation, but we should only user it when it's admin-facing, not user-facing imho.